maven-3.5-jdk8-alpine/library.bash

```diff
@@ -0,0 +1,27 @@
+Maven::Docker_build () {
+  source "$PROJECT_ROOT_DIRECTORY/common/library.bash"
```

Comment:
When sourcing in every function, there is the risk that the library file gets loaded twice (if two functions are called in the same script). If it happens; the script will try to re-define the functions on the second sourcing, and it will fail. Thus, sourcing should be moved in "client" file (`build.bash`, `publish.bash`, ...)

Reply:
Done